llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2403,6 +2403,8 @@ bool LoopAccessInfo::canAnalyzeLoop() {
     return false;
   }
 
+  LLVM_DEBUG(dbgs() << "LAA: Found a loop: " << TheLoop->getHeader()->getName()
```

Comment:
Is this accurate? Might be better to say found an analyzable loop?

Reply:
Unsure if we should change this output to diverge from LV.